runtime/auth/http-auth-aws/common/src/aws/smithy/kotlin/runtime/http/auth/SigV4AsymmetricAuthScheme.kt

```diff
@@ -32,6 +32,19 @@ public class SigV4AsymmetricAuthScheme(
         },
     )
 
+    public constructor(
+        awsSigner: AwsSigner,
+        serviceName: String? = null,
+        clientSigningRegionSet: List<String>? = null,
```

Comment:
naming suggestion: `signingRegionSet`

Reply:
We don't need a new constructor for this do we? Can we not just add a new default param?